__tests__/helpers/index.ts

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright 2022 Mia srl
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import lc39 from '@mia-platform/lc39'
+import { LogLevel } from 'fastify'
+
+import { EnvVariables } from '../../src/env'
+import { MainService } from '../../src/types'
+import { LOG_LEVEL } from '../config'
+
+export const startService = async(conf: EnvVariables): Promise<MainService> => {
+  return (lc39 as unknown as (
```

Comment:
lc39 has no proper typing already? We could fix the library if needed

Reply:
I think it has already been fixed with https://github.com/mia-platform/lc39/pull/198, but it has not been released yet.

Reply:
I've just published v6.0.1 with the fix

Reply:
Latest commit should include the fix